src/components/composites/Button/index.tsx

```diff
@@ -129,16 +128,16 @@ const Button = ({
     ...(outline || transaprent ? { color: variant ?? "indigo.6" } : {}),
   };
 
+  const { icon } = props;
```

Comment:
Destructure icon in the component definition please.

Reply:
Done